pkg/catalogv2/system/system.go

```diff
@@ -399,41 +406,47 @@ func (m *Manager) isInstalled(namespace, name, version, minVersion string, desir
 			return false, "", nil, err
 		}
 
-		desiredValue = map[string]interface{}{}
-		if err := json.Unmarshal(patchedJSON, &desiredValue); err != nil {
+		current, err := semver.NewVersion(r.Chart.Metadata.Version)
+		if err != nil {
 			return false, "", nil, err
 		}
 
-		ver, err := semver.NewVersion(release.Chart.Metadata.Version)
+		desired, err := semver.NewVersion(latestVersion)
 		if err != nil {
 			return false, "", nil, err
 		}
 
+		if minVersion == "" {
```

Comment:
Nit:

This part of the logic makes the exact version overrideable by the min version. While that is the behavior that we want for the webhook's min version, I think that there's an argument to be made that we don't want this to be enforced for everything - and that the webhook should choose, when requesting an install, to request either a min version or exact version install.